util/create_python_wheels.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# creates python wheels for current python version for copying over to internet-disabled machine
+vers=$(python3 --version| awk '{print $NF}')
+wheel_dir="${vers}_wheels"
+python3 -m pip wheel pip -w "$wheel_dir"
+python3 -m pip wheel -r requirements.txt -w "$wheel_dir"
+tar -cvzf "${wheel_dir}.tar.gz" "$wheel_dir"
```

Comment:
Could we prevent permission problems by giving ownership to root or something else? @pmahoney 
```suggestion
tar -cvzf "${wheel_dir}.tar.gz" "$wheel_dir" --owner=0 --group=0
```

Reply:
This looks like a reasonable standard choice here. It's nice to formalize this procedure.